tueplots/figsizes.py

```diff
@@ -1,7 +1,7 @@
 """Figure size settings."""
 
 # Some useful constants
-_GOLDEN_RATIO = (5.0**0.5 - 1.0) / 2.0
+_GOLDEN_RATIO = (5.0 ** 0.5 - 1.0) / 2.0
```

Comment:
Black changed its formatting style IIRC. Do you perhaps have an old version installed? 

```suggestion
_GOLDEN_RATIO = (5.0**0.5 - 1.0) / 2.0
```
should fix it. (I didn't test it.)

Reply:
Yes, I'm using the version we're pinning in ProbNum. I pushed a fix